compiler-rt/lib/scudo/standalone/tests/secondary_test.cpp

```diff
@@ -336,6 +337,37 @@ TEST_F(MapAllocatorCacheTest, CacheOrder) {
     MemMap.unmap();
 }
 
+TEST_F(MapAllocatorCacheTest, PartialChunkHeuristicRetrievalTest) {
+  const scudo::uptr MaxUnusedCacheBytes = scudo::MaxUnusedCachePages * PageSize;
```

Comment:
Don't need to use `scudo::MaxUnusedCachePages`, just pick a constant that makes sense.

Reply:
Ok I just did `PageSize` since `MaxUnusedCacheBytes` shouldn't change the behavior of this test as long as it's page-aligned.